### ISSUE TITLE
Monospace Link Roundtrip

### ISSUE THREAD

**Issue by irmerk** [accordproject/markdown-transform#164]
**Describe the bug**
Monospace links are broken in the roundtrip

**Screenshots**

---

<img width="539" alt="Screen Shot 2020-02-06 at 2 10 42 PM" src="https://user-images.githubusercontent.com/36460856/73970172-af5ee380-48ea-11ea-8225-3aeba0df0713.png">

---

**Additional context**
This may need to be moved to `markdown-editor`?

**Comment by jeromesimeon** [comment]
I'm not sure I understand this issue. What's a monospace link?

**Comment by jeromesimeon** [comment]
> I'm not sure I understand this issue. What's a monospace link?

The commonmark spec has a notion of link, but nothing font-specific. https://spec.commonmark.org/0.29/#links

**Comment by irmerk** [comment]
Now that I think about it, maybe this is more of an issue for the `markdown-editor`. This is what I mean by monospace link:

<img width="1178" alt="Screen Shot 2020-02-10 at 8 55 47 AM" src="https://user-images.githubusercontent.com/36460856/74155829-32ca4e80-4be3-11ea-87ed-4ea3ecda3393.png">

**Comment by jeromesimeon** [comment]
This seems to be parsed and round-trip properly.
```
bash-3.2$ cat codeinlink.md 
this is a [`monospace link`](https://google.com)
bash-3.2$ markus normalize --sample codeinlink.md --output test.md
11:18:44 AM - info: Creating file: test.md
11:18:44 AM - info: 
this is a [`monospace link`](https://google.com "")
```

@jolene, what is the error you are getting?

**Comment by jeromesimeon** [comment]
Some more investigation, the Slate transform does lose the inline code:
```
bash-3.2$ markus normalize --sample codeinlink.md --output test.md --slate 
11:29:28 AM - info: Creating file: test.md
11:29:28 AM - info: 
this is a [monospace link](https://google.com "")
```

**Comment by jeromesimeon** [comment]
This issue isn't specific to code inlines inside links, but to anything other than a text node inside a link.

For instance, with emphasis:
```
bash-3.2$ cat test/data/linkem.md 
this is a [*code link*](https://google.com)
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/linkem.md --slate 
12:20:29 PM - info: 
this is a [](https://google.com "")
```

**Comment by jeromesimeon** [comment]
This should be fixed with #171 

```
bash-3.2$ cat test/data/linkem.md 
this is a [*code link*](https://google.com)
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/linkem.md --slate 
12:21:23 PM - info: 
this is a [*code link*](https://google.com "")
bash-3.2$ more test/data/linkcode.md 
more test/data/linkcode.md 
this is a [`code link`](https://google.com)

this is a `code snippet`
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/linkcode.md --slate 
12:22:25 PM - info: 
this is a [`code link`](https://google.com "")

this is a `code snippet`
```

**Comment by jeromesimeon** [comment]
A more fancy test:
```
bash-3.2$ more test/data/linkmixed.md 
[This is a *link* with `code` and **bold**](http://google.com)
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/linkmixed.md --slate 
12:30:04 PM - info: 
[This is a *link* with `code` and **bold**](http://google.com "")
```
